.github/workflows/unittest.yaml

```diff
@@ -0,0 +1,22 @@
+name: Operator Unit Tests
```

Comment:
As mentioned in the previous PR please look at out [other workflows](https://github.com/canonical/hydra-operator/tree/main/.github) and copy them.